wiki/Tournaments/TCT/v4/en.md

```diff
@@ -0,0 +1,462 @@
+| **Tiger Mafia** | **4** | 0 | 404 enrin not found | [#1](https://osu.ppy.sh/community/matches/101603190) |
+| **Lipton** | **4** | 0 | Dribblers | [#1](https://osu.ppy.sh/community/matches/101603313) |
+| **israel** | **4** | 1 | Foukat cukr do prdele | [#1](https://osu.ppy.sh/community/matches/101603194) |
+| a dom | 3 | **4** | **SwitzerFrance** | [#1](https://osu.ppy.sh/community/matches/101608010) |
+| **not hehe** | **4** | 0 | Hoteddi\_18 | [#1](https://osu.ppy.sh/community/matches/101610813) |
+| **Tiger Mafia** | **4** | 0 | Imagine Playing Basketball with Jesus and Getting Crossed Up | [#1](https://osu.ppy.sh/community/matches/101613192) |
+
+### Qualifiers
+
+Detailed statistics for this round can be found [here](https://docs.google.com/spreadsheets/d/1irT-oEpMu6QfUzKBosyeZJtfKZzcqKmRYAJu9nyWj7s/edit) alongside the final standings on which player made it through to the next stage.
+
+## Ruleset
+
+### General rules
+
+1. A valid team formation will be:
+   - A participant with rank range between #1 and #150
+   - A participant with rank range below #151
+2. Maximum team member per team is **2 players**, meaning there will be no backup or replacement.
+3. The Score System used will be **Score V2**.
+4. Teams will advance through the qualifiers stage first to qualify, the top 24 teams will then proceed to the group stage when they'll get filtered again with top 2 teams per each group will be finally in the knockout stage.
+5. Taiko Convert Tournament v4 will use double-elimination bracket, which means that there will be a loser bracket and winner bracket.
+6. Even in a tournament scene osu! is still a game. Please respect each other to everyone, be friendly, and fair play.
+
+### In-game conditions
+
+#### Qualifiers
+
+1. Every players will be invited at the time to the room.
+2. There will be no warmups, no ban phase, and no pick phase.
+3. In the qualifiers round, each map will be played twice, or you can play once and leave if you think you've set your best score.
+4. There will be a maximum time of **2 minutes** in between maps. The referee will use `!mp start 10` command to start the game.
+5. If a player gets disconnected for some reason in the first 10 seconds of the song, the referee will abort the game and restart it.
+   - If it happens after 10 seconds, the score before the disconnect will be counted.
+6. The referee can penalise teams if any player present bad behaviour in the game room.
+7. Once the match has finished, the room will be completely closed.
+8. If you fail, the score **will still be counted**.
+
+#### Group Stage and Knockouts
+
+1. Every players will be invited at the time to the room.
+2. The referee can wait a maximum of **10 minutes** for each player to attend.
+   - If the player doesn't come, the referee will ask the other team for a 1v2 start. It will depend on the other team if they want to play or not.
```

Comment:
```suggestion
   - If a player does not arrive, the referee will ask the other team for a 1v2 start. It will depend on the other team if they want to play or not.
```